src/OneDrive.Sdk.Authentication.WinStore/OnlineIdAuthenticationProvider.cs

```diff
@@ -38,12 +38,19 @@ public class OnlineIdAuthenticationProvider : MsaAuthenticationProvider
         private readonly OnlineIdAuthenticator authenticator;
         private readonly CredentialPromptType credentialPromptType;
 
+        public enum PromptType
+        {
+            PromptIfNeeded,
```

Comment:
That way it doesn't rely on ordering remaining the same in the `CredentialPromptType` enum